api/server/router/image/image_routes.go

```diff
@@ -45,9 +44,13 @@ func (s *imageRouter) postImagesCreate(ctx context.Context, w http.ResponseWrite
 	version := httputils.VersionFromContext(ctx)
 	if versions.GreaterThanOrEqualTo(version, "1.32") {
 		apiPlatform := r.FormValue("platform")
-		platform = system.ParsePlatform(apiPlatform)
-		if err = system.ValidatePlatform(platform); err != nil {
-			err = fmt.Errorf("invalid platform: %s", err)
+		platform = &specs.Platform{}
```

Comment:
already done in the `var` block

Reply:
Yes, will remove it. Thanks for the comments! ⭕️